# Description: the new gnome-remote-desktop requires the protocol to be
enable by tweaking the corresponding gsettings key
# Forwarded: not-needed, the code is working in the newer version
Index: gnome-control-center/panels/sharing/cc-sharing-panel.c
===================================================================
--- gnome-control-center.orig/panels/sharing/cc-sharing-panel.c
+++ gnome-control-center/panels/sharing/cc-sharing-panel.c
@@ -1094,6 +1094,12 @@ cc_sharing_panel_setup_screen_sharing_di
   gtk_header_bar_pack_start (GTK_HEADER_BAR (self->screen_sharing_headerbar), w);
   self->screen_sharing_switch = w;
 
+  g_settings_bind (vnc_settings,
+                   "enable",
+                   self->screen_sharing_switch,
+                   "active",
+                   G_SETTINGS_BIND_DEFAULT);
+
   cc_sharing_panel_bind_networks_to_label (self, networks,
                                            self->screen_sharing_row);
 }
